test/parallel_api/algorithm/alg.sorting/sort_by_key.pass.cpp

```diff
@@ -82,7 +167,16 @@ main()
     test_with_usm<sycl::usm::alloc::shared>(q);
     // Run tests for USM device memory
     test_with_usm<sycl::usm::alloc::device>(q);
+    // Run tests for sycl buffers
+    test_with_buffers(q);
 #endif // TEST_DPCPP_BACKEND_PRESENT
 
-    return TestUtils::done(TEST_DPCPP_BACKEND_PRESENT);
+#if !TEST_DPCPP_BACKEND_PRESENT
+    test_with_std_pol(oneapi::dpl::execution::seq);
```

Comment:
are we saving 3 symbols?:)

Let's make it `test_with_std_policy`

Reply:
yes, pol is "shorter" then "policy". I prefer short names.
It is not public API, it just a test.